osc-staging.py

```diff
@@ -66,6 +66,8 @@ def _full_project_name(self, project):
               help='force the selection to become a move')
 @cmdln.option('--by-develproject', action='store_true',
               help='sort the packages by devel project')
+@cmdln.option('--supersede', action='store_true',
+              help='superseding the requests. please make sure you have the permission of staging')
```

Comment:
`supersede the request. Needs staging permissions` ?? But not sure, my English sucks